.eslintrc.json

```diff
@@ -0,0 +1,247 @@
+{
```

Comment:
Just a general coment, any reason why we're not doing
```json
{
  "extends": "@decidim"
}
```
as in [decidim/decidim](https://github.com/decidim/decidim/blob/develop/.eslintrc.json)? Is it to have better control on which configurations we use/being able to change whatever we feel to?